scripts/error-codes/codes.json

```diff
@@ -338,5 +338,7 @@
   "337": "An invalid event responder was provided to host component",
   "338": "ReactDOMServer does not yet support the fundamental API.",
   "339": "An invalid value was used as an event responder. Expect one or many event responders created via React.unstable_createResponer().",
-  "340": "An invalid value was used as an event listener. Expect one or many event listeners created via React.unstable_useResponer()."
+  "340": "An invalid value was used as an event listener. Expect one or many event listeners created via React.unstable_useResponer().",
```

Comment:
@trueadm Noticed a typo here. Should be `unstable_useResponder()`

Reply:
I’m on PTO. cc @necolas

Reply:
Oh yeah, sorry. I mean I can do it too :D

Reply:
https://github.com/facebook/react/pull/16373